test-tap/test.js

```diff
@@ -750,6 +750,26 @@ test('timeout is refreshed on assert', t => {
 	});
 });
 
+test('timeout with custom message', t => {
```

Comment:
I'm pushing some code so that we can test this behavior using AVA itself. Don't worry it's still early days for self-hosted tests.

Reply:
Tests are much cleaner that way. Is the `@ava/test` package the ava test runner itself?

Reply:
Glad you like it!

Yes it's a wrapper so we get a `test-ava` bin stub.

Reply:
Nice, thanks!

Reply:
@novemberborn Is there any roadmap to move to move the existing test suit to use self-hosted tests? I quite enjoy working with tests, and would like to help with the migration process, if that is the goal.